spec/readwise/client_spec.rb

```diff
@@ -20,4 +21,15 @@
       subject.export
     end
   end
+
+  context 'retrieving a highlight' do
+    subject { Readwise::Client.new(token: 'foo') }
+    let(:highlight_response) { Readwise::Highlight.new }
+
+    it 'can parse highlight data' do
+      expect(subject).to receive(:get_highlight).and_return(highlight_response)
+
+      subject.get_highlight
+    end
+  end
```

Comment:
Hopefully this is within the realm of what's needed

Reply:
I think we can do better :) 

Currently, this test doesn't do anything - it stubs the `get_highlight` method and a dummy return value and then calls that stubbed method.

I think it would be better to stub out the underlying API call with a mocked HTTP body/response (e.g. https://github.com/andjosh/readwise-ruby/blob/054f040acccdce26ef52d9fbd250995918b1b313/spec/readwise/client_spec.rb#L13-L22) and then test that:
- the method does the validation that a highlight ID is passed as an argument
- The method actually makes the intended underlying API call
- The method correctly parses the HTTP response body and returns an instance of `Highlight`

Reply:
I think what I just pushed accomplishes at least a bit of what you prescribed, thank you for the suggestions! Writing good tests is something I'm definitely still working on.

Reply:
Thanks!